users/factories.py

```diff
@@ -30,6 +30,15 @@ class Meta:
     is_active = True
     preferences = factory.RelatedFactory(UserPreferencesFactory, factory_related_name="user")
     password = make_password(FACTORY_USER_PASSWORD)
+    need_change_password = False
+
+    @factory.post_generation
+    def set_password(  # type: ignore
+        self, create, extracted, **kwargs
+    ):  # pylint: disable=unused-argument, no-value-for-parameter
+        self.set_password("password")
+        if create:
+            self.save()
```

Comment:
I don't think we need it here. We're already handling setting password above in 32 line

Reply:
That's true, actually I didn't write it, but I've missed that Marcin fixed that.